src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -15,19 +13,29 @@
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.Numeric;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
```

Comment:
Ctrl+K+G

Reply:
sorted them

---
In reply to: [227047810](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227047810) [](ancestors = 227047810)